tests/cpp-tests/Classes/AppDelegate.cpp

```diff
@@ -165,12 +165,7 @@ bool AppDelegate::applicationDidFinishLaunching()
     
     fileUtils->setSearchPaths(searchPaths);
 
-#if (CC_TARGET_PLATFORM == CC_PLATFORM_WP8)
-    // a bug in DirectX 11 level9-x on the device prevents ResolutionPolicy::NO_BORDER from working correctly
     glview->setDesignResolutionSize(designSize.width, designSize.height, ResolutionPolicy::SHOW_ALL);
```

Comment:
There was a bug in the old WP8 GPU drivers that returned incorrect values when using negative positions for the viewport as calculated by cocos2d-x.